### ISSUE TITLE
Not working with empty formName() and one column

### ISSUE THREAD

**Issue by olegbaturin** [unclead/yii2-multiple-input#347]
[Here](https://github.com/unclead/yii2-multiple-input/blob/fa4a69a96643b532ea2b49f86bdd6aeb6cd201a7/src/MultipleInputColumn.php#L90) you consider that Model::formName() returns not empty string. As a result input names look like "[attribute][index]" because [here](https://github.com/unclead/yii2-multiple-input/blob/fa4a69a96643b532ea2b49f86bdd6aeb6cd201a7/src/MultipleInputColumn.php#L53) you consider that prefix can't be empty.

**Comment by unclead** [comment]
How to reproduce the issue? In other words, post the code snippet to reproduce the issue

**Comment by olegbaturin** [comment]
```
class MyForm extends \yii\base\Model

public function formName()
{
    return '';
}
```

**Comment by olegbaturin** [comment]
Use [example](https://github.com/unclead/yii2-multiple-input/wiki/Usage#one-column) for single column. Or set the name for only one column the same as the attribute name. See [this](https://github.com/unclead/yii2-multiple-input/blob/fa4a69a96643b532ea2b49f86bdd6aeb6cd201a7/src/MultipleInputColumn.php#L89) condition.

**Comment by unclead** [comment]
Any suggestions on how to fix it? 😄  Throw an exception if form name is empty?

**Comment by olegbaturin** [comment]
formName() CAN return empty string! It's just a prefix for input name. With prefix name will be "MyForm[email]" (by default), without "email".


> Any suggestions on how to fix it?

Always [return](https://github.com/unclead/yii2-multiple-input/blob/fa4a69a96643b532ea2b49f86bdd6aeb6cd201a7/src/MultipleInputColumn.php#L93) default prefix and check [this](https://github.com/unclead/yii2-multiple-input/blob/fa4a69a96643b532ea2b49f86bdd6aeb6cd201a7/src/MultipleInputColumn.php#L89) condition [here](https://github.com/unclead/yii2-multiple-input/blob/fa4a69a96643b532ea2b49f86bdd6aeb6cd201a7/src/MultipleInputColumn.php#L52). So [this](https://github.com/unclead/yii2-multiple-input/blob/fa4a69a96643b532ea2b49f86bdd6aeb6cd201a7/src/MultipleInputColumn.php#L53) name will be "'[' . $this->name . '][' . $index . ']'" or "'[' . $index . ']'"
I think.

**Comment by olegbaturin** [comment]
Do you accept a PRs?

**Comment by unclead** [comment]
I would rephrase - at the moment I only rely on PR 😄

**Comment by unclead** [comment]
@olegbaturin  so will you make a PR or it was a rhetorical question? 😄

**Comment by olegbaturin** [comment]
I'll have some time to this fix in the next week or so.
